synapse/rest/client/v2_alpha/relations.py

```diff
@@ -153,6 +153,12 @@ def on_GET(self, request, room_id, parent_id, relation_type=None, event_type=Non
         from_token = parse_string(request, "from")
         to_token = parse_string(request, "to")
 
+        # Check if the event is redacted, and if so return an empty chunk
+        # list and zero tokens
+        if "redacted_because" in event.unsigned:
```

Comment:
I wasn't sure either, but it is used elsewhere. https://github.com/matrix-org/synapse/blob/32e7c9e7f20b57dd081023ac42d6931a8da9b3a3/tests/storage/test_redaction.py#L135

Might be best to create a method `is_event_redacted`, though not sure how it would check that otherwise.

Reply:
This has now been replaced with `internal_metadata`.